src/fcm/index.js

```diff
@@ -2,35 +2,77 @@ const crypto = require('crypto');
 const request = require('request-promise');
 const { escape } = require('../utils/base64');
 
-const FCM_SUBSCRIBE = 'https://fcm.googleapis.com/fcm/connect/subscribe';
+const FIREBASE_INSTALLATION =
+  'https://firebaseinstallations.googleapis.com/v1/';
+const FCM_REGISTRATION = 'https://fcmregistrations.googleapis.com/v1/';
 const FCM_ENDPOINT = 'https://fcm.googleapis.com/fcm/send';
 
-module.exports = registerFCM;
+module.exports = { installFCM, registerFCM };
 
-async function registerFCM({ senderId, token }) {
+function generateFirebaseFID() {
+  // A valid FID has exactly 22 base64 characters, which is 132 bits, or 16.5
```

Comment:
Could we update this comment to elaborate on *why* we only do 17?

Reply:
I do not know why 😅 , this is from the forked repo